.gitignore

```diff
@@ -1,149 +1,274 @@
-# Visual Studio
-*.suo
-*.user
-*.sln.docstates
-*.pch
-*.tlog
-*.log
-*.scc
-*.exp
-*.ilk
-*.lastbuildstate
-vc100.pdb
-ipch
-*.sdf
-*.opensdf
-*.idb
-*.sln
-*.recipe
-
-*.lib
-*.a
-*.obj
-*.dylib
-*.xz
-
-# OSX/Linux build products
-*.mak
-*.mak.vpc_crc
-*.xcodeproj/
-obj*/
-!devtools/*.mak
-!utils/smdlexp/smdlexp.mak
-*.project
-/devtools/bin/vpc_linux

```

Comment:
VPC2 will not be merged

Reply:
I'll work on reverting to vpc.

Reply:
Currently, vpc segfaults on:
```* thread #1, name = 'vpc', stop reason = signal SIGSEGV: invalid address (fault address: 0x55666f24)
    frame #0: 0x00005555555dbce5 vpc`CUtlMemoryPool::AddNewBlob(this=0x0000555555652d28) at mempool.cpp:223:14
   220 		void **newBlob = (void**)m_pHeadOfFreeList;
   221 		for (int j = 0; j < nElements-1; j++)
   222 		{
-> 223 			newBlob[0] = (char*)newBlob + m_BlockSize;
   224 			newBlob = (void**)newBlob[0];
   225 		}
   226 	```
Specifically, on the assignment.

Reply:
Fixed, vpc2 is removed and vpc1 is now default.